src/coreclr/src/vm/class.h

```diff
@@ -1297,6 +1297,18 @@ class EEClass // DO NOT CREATE A NEW EEClass USING NEW!
         LIMITED_METHOD_CONTRACT;
         m_VMFlags |= VMFLAG_LAYOUT_DEPENDS_ON_OTHER_MODULES;
     }
+
+    inline BOOL HasLayoutDependsOnOtherVersionBubbles()
```

Comment:
Do we really need both `HasLayoutDependsOnOtherModules` and `HasLayoutDependsOnOtherVersionBubbles` ?

I would expect to have just the version bubble one.